build-support/settings/README.md

```diff
@@ -0,0 +1,4 @@
+# Redwood Settings Plugin
+
+This is a Gradle plugin applied to the root `settings.gradle` via `includeBuild`
+which automatically adds the `build-support/` Gradle plugin to all projects.
```

Comment:
Eventually this can likely live within the `build-support/` plugin itself. The reason it doesn't today is that use in the `settings.gradle` via the `plugins {}` DSL moves the plugin and its dependencies off of the buildscript classpath and onto the plugin classpath which breaks some things. If the main build eventually migrates to `plugins {}` then it will be possible to move.